pages/glossary/aria-attribute-set.md

```diff
@@ -0,0 +1,30 @@
+---
+title: ARIA state or property is set
+key: aria-attribute-set
+unambiguous: true
+objective: true
+input_aspects:
+  - Accessibility tree
+  - CSS styling
+  - DOM tree
+---
+
+An ARIA [state][aria state] or [property][aria property] is <dfn>set</dfn> on an [HTML element][namespaced element] when it has a value. This may happen in three ways:
```

Comment:
The term "value" brings to mind the explicit value of an attribute. What if we replace it with "when the accessibility tree exposes its value."